consulService.go

```diff
@@ -35,6 +35,7 @@ type consulServiceState struct {
 	Port        int           // Port number where its listening
 	SessionID   string        // session id assigned by consul
 	stopChan    chan struct{} // Channel to stop ttl refresh
+	Hostname    string        // Host name where its running
```

Comment:
Looks like this field is not being set. Can you please set it in RegisterService()?